docs/SETUP.md

```diff
@@ -0,0 +1,227 @@
+# Quickstart
+
+Make a copy of `.env_sample` called `.env` in the root of the project directory. Confirm settings,storage,SSL and ports.
+
+Run the dev_setup script to install all dependencies and create a new virtual environment (venv) for CodaLab:
+
```

Comment:
What's the dev_setup script commands for mac? If Mac doesn't need one, then we should specify that.